types/nationalAvalancheCenter/schemas.ts

```diff
@@ -17,6 +17,8 @@ export const avalancheCenterIDSchema = z.enum([
   'TAC', // Taos: NM
   'WAC', // Wallowas: OR
   'WCMAC', // West Central Montana: MT
+  'CNFAIC', // Chugach : AK
```

Comment:
@stevekuznetsov these are special case ones, right? they were in the link of urls but not in here, I assume we want to add them?

Reply:
I think you need to rebase, they are now in the types.